source/clear-linux/get-started/live-image.rst

```diff
@@ -1,7 +1,7 @@
 .. _live-image:
 
-Live Image Installation
-#######################
+Install Clear Linux from a live image
```

Comment:
I would consider changing it to: as a live image
You don't install CL from the live image, you run it from it.